adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostBannerAd.m

```diff
@@ -0,0 +1,176 @@
+// Copyright 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "GADMAdapterChartboostBannerAd.h"
+
+#import <Chartboost/Chartboost+Mediation.h>
```

Comment:
Can we substitute this import by this?
```
#if __has_include(<Chartboost/Chartboost+Mediation.h>)
#import <Chartboost/Chartboost+Mediation.h>
#else
#import "Chartboost+Mediation.h"
#endif
```
It helps with our testing process by making mocking Chartboost headers easier.